pylint/pyreverse/inspector.py

```diff
@@ -80,6 +80,35 @@ def generate_id(self):
         return self.id_count
 
 
+class Project:
```

Comment:
Moved this because of typing on line 116. Also added `self.uid = None`

Reply:
Instead of moving it, just use a string as type annotation -> `node: "Project"`

Reply:
Don't you think moving it is better? That way we don't define before accessing.